config/Readme.md

```diff
@@ -1,3 +1,14 @@
 ## Config
 
-All config files must be in this folder. If there is no option to set this folder directly, it has to be hardlinked.
\ No newline at end of file
+All config files must be in this folder. If there is no option to set this folder 
+directly, it has to be hardlinked.
+
+* `aliases`: aliases in cmd; called form vendor\init.bat; autocreated from 
+  `vendor\aliases.example`.
+* `*.lua`: clink completitions and prompt filters; called from vendor\cmder.lua after all 
+  other prompt filter and clink completitons are initialized; add your own.
+* `user_profile.{sh|bat|ps1}: startup files for bash|cmd|powershell tasks; called from their 
+  respective startup scripts in `vendor\`; autocreated on first start of such a task
+* `.history`: the current commandline history; autoupdated on close
+* `settings`: settings for readline; overwritten on update
```

Comment:
Someone should review the last two lines (settings and conemu.xml)... This is just my current understanding of this...